src/main/java/domain/car/Car.java

```diff
@@ -0,0 +1,51 @@
+package domain.car;
+
+import domain.strategy.Movable;
+import domain.strategy.RandomMoveStrategy;
+
+public class Car {
+    private final String carName;
+    private int position;
+    private final Movable movable;
+    private static final int MAX_LENGTH = 5;
```

Comment:
클래스에서 필드, 메서드 등 여러 요소의 선언에도 순서가 존재하는 것을 알고 계신가요?

사용하는 구글 컨벤션에서는 정답이 없다고 가이드하고 있어요.

https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents

하지만 Oracle에서는 다음과 같이 순서를 가이드하고 있어요.

https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html#1852

이를 지킬 필요는 없지만, 추후 혼란을 막기 위해 이러한 선언 순서에도 신경을 쓰는 것을 추천드려요.

(저는 상수는 맨 위에 선언하고, 한 칸 띄우는 것을 선호합니다)

```suggestion
    private static final int MAX_LENGTH = 5;
    
    private final String carName;
    private int position;
    private final Movable movable;
```

Reply:
오오 순서가 있으면 좋을 것 같다고 생각은 했는데 오라클에서는 가이드가 있군요! 
상수 선언 뒤 한 줄 띄우니 훨씬 보기 편하네요😮 저도 적용해봐야겠습니다!